README.md

```diff
@@ -8,25 +8,67 @@ Digital Content Store Project (Semester 2, 2014/2015).
 
 ## Setup
 
+* Python 3
+* MySQL Server
+
 ```
+# Clone Project
+git clone git@github.com:exclusive-gix/cs2102-store.git dcs
+cd dcs
 
 # Install requirement
 pip install -r requirements.txt
 
+# Configure Project
+cp dcs/__local_settings.py dcs/local_settings.py
```

Comment:
because everyone of us have different django configuration
Such as:
- `DATABASE`: user, password, schema, etc...
- `SECRET_KEY`
- `DEBUG`
- `EMAIL_BACKEND`
- `STATIC_ROOT`
- `STATIC_URL`
- etc...